toolkit/imageconfigs/packagelists/core-tools-packages.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need to add "KernelCommandLine": { "SELinux": "permissive" } here?

Reply:
In core-tools-packages.json? Not required.